src/5e-SRD-Monsters.json

```diff
@@ -17235,7 +17445,26 @@
     "actions": [
       {
         "name": "Multiattack",
-        "desc": "The grick makes one attack with its tentacles. If that attack hits, the grick can make one beak attack against the same target."
+        "desc": "The grick makes one attack with its tentacles. If that attack hits, the grick can make one beak attack against the same target.",
+        "options": {
+          "choose": 1,
+          "from": [
+            [
+              {
+                "name": "Tentacles",
+                "count": 1,
+                "type": "melee"
+              }
+            ],
+            [
+              {
+                "name": "Beak",
```

Comment:
We should probably add a `note` field here as well since this only works if the first one hits.